scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -863,6 +867,13 @@ function Setup-Environment {
             'See https://aka.ms/iotedge-platsup for more details.')
         $preRequisitesMet = $false
     }
+    
+    if (Test-IoTCore) {
+        if (-not (Test-Path $ContainersFeatureRegPath) -or (-not [bool] (Get-Package $ContainersFeaturePackageName)) -or (-not [bool] (Get-Package $ContainersFeatureLangPackageName))) {
+            Write-HostRed "The container host does not have 'Containers Feature' enabled"
```

Comment:
Can you also print what the user should do to install it? `Please run "Install-Package $ContainersFeaturePackageName"` or so.

Reply:
User must add Containers package during Iot Core image creation process. Added message for that.

Reply:
Fixed.